horizons/gui/gui.py

```diff
@@ -78,6 +80,13 @@ def on_escape(self):
 		if self._windows.open_popup(_("Quit Game"), message, show_cancel_button=True):
 			horizons.main.quit()
 
+	def mouse_entered_changebackground(self):
+		changebackground = self._gui.findChild(name='changeBackground')
+		changebackground.background_color = (0, 0, 0, 255)
+
+	def mouse_exited_changebackground(self):
+		changebackground = self._gui.findChild(name='changeBackground')
+		changebackground.background_color = (0, 0, 0, 102)
```

Comment:
If this is defined in the corresponding XML file, it should be taken from there or saved in `mouse_entered_changebackground` and replaced here. It should not be hard-coded at two different places.  Otherwise, some future developer might be confused why it doesn't work.